### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by willmcgugan**
**What this PR solves**

- Lack of docs.
- Some name changes and repr changes.

**How it is done**

- Wrote docs.

**What to look out for**

- Bad spelling. Grammar. Naughty words. 

**Review**

- [X] Ready for review

### CODE REVIEW

File: lomond/constants.py [hunk 1]
```diff
@@ -11,10 +13,12 @@
...
 WS_VERSION = 13
 
-# Only report major.minor in version
-_version_identifier = '.'.join(__version__.split('.')[:2])
-
 # User agent sent with websocket request
-# Default will be DataplicityLomond/<major>.<minor
-# e.g "DataplicityLomond/0.1"
-USER_AGENT = 'DataplicityLomond/{}'.format(_version_identifier)
+USER_AGENT = 'Lomond/{} {}/{} {}/{} ({})'.format(
```
Comment (willmcgugan) [inline lomond/constants.py]: This makes a fairly detailed user agent. Here is an example:

`Lomond/0.1.7 CPython/2.7.13 Darwin/16.6.0 (Darwin-16.6.0-x86_64-i386-64bit)`

**Comment by willmcgugan** [comment]
@wildfoundry/web-developers Try not to be *too* pedantic re grammar. As long as this is factually correct, I'll edit in another PR.

File: lomond/constants.py (hunk 1 above)
Comment (toudi) [inline lomond/constants.py]: could we have an additional placeholder here? What I mean is that if we have the agent, wormhole, ttng and router all using lomond then how do we filter out which client in particular is sending requests? if we would have additional field, we could assign some nice name like

```session = WebsocketSession('ws://example.com', client='dataplicity-agent')```

which would yield

```Lomond/0.1.7 CPython/2.7.13 Darwin/16.6.0 (Darwin-16.6.0-x86_64-i386-64bit dataplicity-agent```

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+=======
+
+Lomond was sponsored by `Dataplicity <https://www.dataplicity.com/>`_ --
+Control your Raspberry Pi from anywhere!
+
+Authors
+=======
+
+- Will McGugan (willmcgugan+lomond@gmail.com)
+- Mateusz Mikołajczyk
```
Comment (toudi) [inline docs/index.rst]: I merely wrote the tests, I am definetely not the author ;)

File: lomond/constants.py (hunk 1 above)
Comment (willmcgugan) [inline lomond/constants.py]: I added an `agent` parameter to the WebSocket object. I'd assume people would do something like:

```
WebSocket('wss://example.org', agent="MyClient/0.1.0 " + constants.USER_AGENT)
```

To be honest, I think a `client` parameter may be too much hand-holding.

File: docs/index.rst (hunk 2 above)
Comment (willmcgugan) [inline docs/index.rst]: You authored the tests!

File: lomond/constants.py (hunk 1 above)
Comment (toudi) [inline lomond/constants.py]: ah, this settles it then. I was talking about exactly such a thing

**Pull request merged**